src/vcpkg/commands.applocal.cpp

```diff
@@ -0,0 +1,537 @@
+                                                 "Qt5QuickTemplates2d.dll",
+                                                 "Qt5QmlWorkerScript.dll",
+                                                 "Qt5QmlWorkerScriptd.dll",
+                                                 "Qt5QuickParticles.dll",
+                                                 "Qt5QuickParticlesd.dll",
+                                                 "Qt5QuickWidgets.dll",
+                                                 "Qt5QuickWidgetsd.dll"};
+                for (std::string lib : libs)
+                {
+                    deploy_binary(target_binary_dir, bin_dir, lib);
+                }
+
+                deployPluginsQt("scenegraph", target_binary_dir, qt_plugins_dir);
+                deployPluginsQt("qmltooling", target_binary_dir, qt_plugins_dir);
+            }
+
+            std::error_code ec;
+            std::vector<Path> children = m_fs.get_files_non_recursive(target_binary_name, ec);
+            for (auto&& c : children)
+            {
+                if (c == "Qt5Declarative.dll" || c == "Qt5Declaratived.dll")
+                {
+                    deployPluginsQt("qml1tooling", target_binary_dir, qt_plugins_dir);
+                }
+
+                if (c == "Qt5Positioning.dll" || c == "Qt5Positioningd.dll")
+                {
+                    deployPluginsQt("position", target_binary_dir, qt_plugins_dir);
+                }
+
+                if (c == "Qt5Location.dll" || c == "Qt5Locationd.dll")
+                {
+                    deployPluginsQt("geoservices", target_binary_dir, qt_plugins_dir);
+                }
+
+                if (c == "Qt5Sensors.dll" || c == "Qt5Sensorsd.dll")
+                {
+                    deployPluginsQt("sensors", target_binary_dir, qt_plugins_dir);
+                    deployPluginsQt("sensorgestures", target_binary_dir, qt_plugins_dir);
+                }
+
+                if (c == "Qt5WebEngineCore.dll" || c == "Qt5WebEngineCored.dll")
+                {
+                    deployPluginsQt("qtwebengine", target_binary_dir, qt_plugins_dir);
+                }
+
+                if (c == "Qt53DRenderer.dll" || c == "Qt53DRendererd.dll")
+                {
+                    deployPluginsQt("sceneparsers", target_binary_dir, qt_plugins_dir);
+                }
+
+                if (c == "Qt5TextToSpeech.dll" || c == "Qt5TextToSpeechd.dll")
+                {
+                    deployPluginsQt("texttospeech", target_binary_dir, qt_plugins_dir);
+                }
+
+                if (c == "Qt5SerialBus.dll" || c == "Qt5SerialBusd.dll")
+                {
+                    deployPluginsQt("canbus", target_binary_dir, qt_plugins_dir);
+                }
+            }
+        }
+
+        bool deploy_binary(const Path& target_binary_dir,
+                           const Path& installed_dir,
+                           const std::string& target_binary_name)
```

Comment:
General suggestion: whenever you would write `const std::string&` as a function parameter, consider if `StringView` would be appropriate instead.